### DISCUSSION TITLE
Article Font Readability

### DISCUSSION THREAD

**Discussion by GreatScott000** [microsoft/alguidelines#59] [General]
I find the font used on the articles to be difficult to read.  I don't know if it is the colour scheme or the "weight" of the font but I find the text "bleeds" into the background and is almost fuzzy.  Anyone else have this problem?  Or is it just my old eyesight?
<img width="957" alt="image" src="https://user-images.githubusercontent.com/4333956/145016452-3def705d-f600-45c2-8d98-4518484a2174.png">

**Comment by NKarolak** [comment]
I understand what you are meaning. I think, it is the combination of the font (very narrow lines), dark gray font color plus the light gray background color. Too little contrast.

**Comment by GreatScott000** [comment]
I tried to switch to Dark Mode to see if that would help but it does not seem to affect the page(s) at all.

**Comment by waldo1001** [comment]
@JeremyVyska - what do you think?

**Comment by JeremyVyska** [comment]
Gah.  Dug into the Theme's CSS.  Font weight 300?   Blech.

Difference example:
![image](https://user-images.githubusercontent.com/35526546/145024981-9e933f7b-5511-4e25-8ddb-ca8cf9fbc3bb.png)

I'll start there, but might warrant some workshopping.

**Reply by GreatScott000** [reply]
Better but only slightly.  Still "fuzzy".

Appreciate the support in looking at this.

**Reply by NKarolak** [reply]
What would be bad about a plain black font color instead?

**Reply by JeremyVyska** [reply]
I am for tackling both.

**Comment by JeremyVyska** [comment]
@GreatScott000 - hopefully new theme resolves a bunch of these.

**Reply by rsaritzky** [reply]
For text, especially in the context of this site, I don't see a good reason to use a light grey color.  Bold?  YES  Bright colors? YES

**Comment by GreatScott000** [comment]
Much better thanks.
